composer.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "moxie-leean/wp-elements",
```

Comment:
As was discuss on Tuesday we are going to use `Lean` instead of `Leean` if we can start doing that for the new modules it would be great, we are going to update the previous modules as well.